components/contribution-flow/StepPayment.js

```diff
@@ -189,7 +206,8 @@ class StepPayment extends React.Component {
       subtitle: getPaymentMethodMetadata(pm),
       icon: getPaymentMethodIcon(pm, collective),
       paymentMethod: pm,
-      disabled: pm.balance < minBalance,
+      isExpired: paymentMethodExpired(pm.expiryDate),
+      disabled: pm.balance < minBalance || paymentMethodExpired(pm.expiryDate),
```

Comment:
Maybe we can store the result of `paymentMethodExpired` in a variable so that we don't have to call the function multiple times?

Reply:
I couldn't find a good place to place it. Any suggestions?

Reply:
Just above, here?

![image](https://user-images.githubusercontent.com/1556356/72256681-09c99480-360a-11ea-9057-51e7df65da1a.png)

Reply:
`paymentMethodOptions` is an array of `paymentMethods`... I don't see how this would work

Reply:
```es6
    const paymentMethodsOptions = (collective.paymentMethods || []).map(pm => {
      const isExpired = paymentMethodExpired(pm.expiryDate)
      return ({
        (...),
        isExpired
      })
    });
```